### PR TITLE
[CPSRV-30074] Add Advenir client

### PR DESCRIPTION
**Pull request by aaggarwal-cpi**

### CODE REVIEW

File: build/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ ! -f /app/properties/base_constant.ini ]; then
+    tar -xzf /etc/generated-secrets/generated-secrets.tar.gz properties/base_constant.ini
+fi
+
+LOG_PATH=/logs/golang-gateway/$(date +%F)/$(echo $POD_NAME | rev | cut -d'-' -f 1-2 - | rev | tr '-' '/')
+
```
Comment (rmak-cpi) [inline build/entrypoint.sh]: golang-gateway -> event-dispatcher

File: build/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env groovy
+pipeline {
+    agent {
+        label 'agent-with-php-and-docker'
+    }
+    parameters {
+        string(name: 'sha1', defaultValue: 'master', description: 'commit to checkout')
+        string(name: 'aws_account_id', defaultValue: '992211689352', description: 'AWS Account ID')
+        string(name: 'ecr_region', defaultValue: 'us-west-2', description: 'ECR region')
+        string(name: 'ecr_repository_name', defaultValue: 'event-dispatcher', description: 'ECR repository name')
```
Comment (rmak-cpi) [inline build/Jenkinsfile]: I went ahead and created the event-dispatcher repository in QA and Prod ECR.

File: build/entrypoint.sh (hunk 1 above)
Comment (aaggarwal-cpi) [reply build/entrypoint.sh]: Done

File: build/Jenkinsfile (hunk 2 above)
Comment (aaggarwal-cpi) [reply build/Jenkinsfile]: Awesome. Thanks. What is the next step to create build pipeline using this jenkinsfile

**Comment by aaggarwal-cpi** [comment]
@rmak-cpi 
Both 1 and 2 are done. Please review again.

**Pull request merged**